src/reference/modules/usage/pages/configuration_management.adoc

```diff
@@ -227,6 +227,25 @@ Based on these facts, the compliance of a rule is calculated like this:
 Number of nodes for which conformity is reached for every directive of the
 rule / Total number of nodes on which the rule has been applied
 
+== Rudder score
+
+Nodes have an individual score, represented by a letter from F (worst) to A (best).
+This score aims at aggregating information from various sources to give an overall
+view of its state.
+
+The node score is computed from several sub-scores:
+
+* The compliance score. It is computed based on the policies' compliance score, associating the score with a compliance percentage.
+* The system updates score. It is based on the number of available upgrades, with a special focus on security upgrades. It represents the score of the patch management for this node.
+* The vulnerability score. Complementary to the available updates scores, it shows the impact of current known vulnerabilities in installed packages. The score is computed using the number of vulnerabilities and their score.
```

Comment:
Maybe we should specify that CVE plugin is required for this score

Reply:
yes